drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -179,6 +179,12 @@ matterManufacturer:
     vendorId: 0x1339
     productId: 0x0040
     deviceProfileName: plug-binary
+#Innovation Matters
+  - id: "4978/1"
+    deviceLabel: M2D Bridge
```

Comment:
Is this label correct? It says this is a bridge but it uses a colorTemp bulb profile? I think this could be confusing for users, unless I misunderstand what this device is.

Reply:
I looked it up and it is indeed a bridge for lights specifically ([here](https://www.innovation-matters.at/en/product-2/m2d_bridge/)) So I'm not sure this is the correct profile it should be using. However, I may just be misunderstanding something